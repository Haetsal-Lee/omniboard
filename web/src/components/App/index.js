import React, { Component } from 'react';
import { Navbar } from 'react-bootstrap';
import RunsTable from '../RunsTable/runsTable';
import { ToastContainer } from 'react-toastify';
import 'react-toastify/dist/ReactToastify.min.css';
import './style.scss';

class App extends Component {
  render() {
    return (
      <div className="App">
        <Navbar inverse fluid>
          <Navbar.Header>
            <Navbar.Brand>
              <a href="/">Omniboard</a>
            </Navbar.Brand>
          </Navbar.Header>
        </Navbar>
        <div className="content">
          <ToastContainer autoClose={false}/>
          <RunsTable/>
        </div>
      </div>
    );
  }
}

export default App;
