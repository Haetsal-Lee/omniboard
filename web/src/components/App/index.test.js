import React from 'react';
import App from './index';

describe('App component', () => {
  it('should render', () => {
    const wrapper = shallow(<App/>);

    expect(wrapper).toMatchSnapshot();
  });
});