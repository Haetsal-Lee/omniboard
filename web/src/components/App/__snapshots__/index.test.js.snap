// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render 1`] = `
<div
  className="App"
>
  <Uncontrolled(Navbar)
    bsStyle="default"
    fluid={true}
    inverse={true}
  >
    <NavbarHeader>
      <NavbarBrand>
        <a
          href="/"
        >
          Omniboard
        </a>
      </NavbarBrand>
    </NavbarHeader>
  </Uncontrolled(Navbar)>
  <div
    className="content"
  >
    <ToastContainer
      autoClose={false}
      bodyClassName={null}
      className={null}
      closeButton={
        <CloseButton
          ariaLabel="close"
        />
      }
      closeOnClick={true}
      draggable={true}
      draggablePercent={80}
      hideProgressBar={false}
      newestOnTop={false}
      pauseOnFocusLoss={true}
      pauseOnHover={true}
      position="top-right"
      progressClassName={null}
      rtl={false}
      style={null}
      toastClassName={null}
      transition={[Function]}
    />
    <RunsTable
      localStorageKey="runsTable"
      stateFilterKeys={
        Array [
          "dropdownOptions",
          "columnOrder",
          "columnWidths",
          "defaultSortIndices",
          "sortIndices",
          "sort",
        ]
      }
    />
  </div>
</div>
`;
